pkg/monitor/datapath_trace.go

```diff
@@ -43,6 +42,7 @@ const (
 )
 
 // TraceNotifyV0 is the common message format for versions 0 and 1.
+// This struct needs to be kept in sync with the function below.
```

Comment:
I would prefer to refer to the function explicitely by name so that moving code around would not "break the comment".
```suggestion
// This struct needs to be kept in sync with the decodeTraceNotifyVersion0
// func.
```